CharacterMap/CharacterMap/Provider/SQLiteGlyphProvider.cs

```diff
@@ -20,7 +20,7 @@ public partial class SQLiteGlyphProvider : IGlyphDataProvider
         internal const string MDL2_SEARCH_TABLE = "mdl2search";
         internal const string FONTAWESOME_SEARCH_TABLE = "fontawesomesearch";
         internal const string UNICODE_SEARCH_TABLE = "unicodesearch";
-        const int SEARCH_LIMIT = 10;
+        private int SEARCH_LIMIT = new AppSettings().MaxSearchResult;
```

Comment:
I'm not sure if this is the best practice to get a setting here. But at the same time, I'm afraid to add AppSettings() on this class because iit probably use once?